CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
 # Azure Kubernetes Service Changelog
 
+## Release Notes 2025-10-12
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* AKS is blocking creation of new clusters using Availability Sets (VMAS) effective September 30, 2025. Existing VMAS clusters must migrate to Virtual Machine Scale Sets (VMSS) before the retirement date. For more information, see [Availability Sets retirement](https://learn.microsoft.com/azure/aks/availability-sets-on-aks#availability-sets-retirement-date).
```

Comment:
>AKS is blocking creation of new clusters using Availability Sets (VMAS) effective September 30, 2025. Existing VMAS clusters must migrate to Virtual Machine Scale Sets (VMSS) before the retirement date

we are already past September 30th. we need to update this line on if this was retired and any behavior/migration users need to be aware of or action on now

Reply:
Lets remove this.